app/destination/page.js

```diff
@@ -1,54 +1,60 @@
 "use client";
 
-import { useState } from 'react';
-
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
-
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
-
+import { useState } from "react";
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import PlanetWishlistItem from "./PlanetWishlistItem";
+import { PlanetCard } from "./PlanetCard";
 
 export const Destinations = () => {
+  //state
   const [selectedPlanets, onAddPlanet] = useState([]);
-
-  let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  const [numberOfPlanets, setNumberOfPlanets] = useState(0);
+  const [isPlanetSelected, setIsPlanetSelected] = useState(false);
```

Comment:
It seems like later you are converting this to an object. So the starting value should be an object too.

Reply:
?. i will research more about this.